### ISSUE TITLE
model_triplot doesn't work with patchwork 1.0.1

### ISSUE THREAD

**Issue by kasiapekala** [ModelOriented/triplot#16]
model_triplot rises error after upgrading patchwork to version 1.0.1
workaround will be introduced shortly in triplot 1.3

**Comment by kasiapekala** [comment]
Temporary workaround - downgrading patchwork to 1.0.0:
```
library(devtools) 
install_version("patchwork", version = "1.0.0")
```

**Comment by kasiapekala** [comment]
fixed in triplot 1.3 (to be send to CRAN)
